html/recordplaytest.html

```diff
@@ -89,7 +89,11 @@ <h3 class="panel-title">Recorder/Playout</h3>
 					<div class="col-md-6 hide" id="video">
 						<div class="panel panel-default">
 							<div class="panel-heading">
-								<h3 class="panel-title"><span id="videotitle">Remote Video</span> <button class="btn-xs btn-danger pull-right" autocomplete="off" id="stop">Stop</button></h3>
+								<h3 class="panel-title">
+									<span id="videotitle">Remote Video</span>
+									<button class="btn-xs btn-danger pull-right" autocomplete="off" id="stop">Stop</button>
+									<button class="btn-xs btn-primary pull-right" autocomplete="off" id="pause-resume">Pause</button>
```

Comment:
This button is currently visible also when playing back recordings, which shouldn't happen (especially since it does nothing in that case).